packages/table/components/Table.tsx

```diff
@@ -38,7 +38,18 @@ export interface TableProps {
   /**
    * Selected row data
    */
-  selectedRows?: object[];
+  selectedRows?: {};
+  /**
+   * Set the row height
+   */
+  rowHeight?: number;
```

Comment:
I'm not sure how this got in here. This is from this PR: https://github.com/dcos-labs/ui-kit/pull/294

I have no problem with specifying a set of options instead of letting it be any number, but I would do it in the PR I linked above and remove it entirely from this one